subgraph/src/mappings/vault.ts

```diff
@@ -89,7 +89,7 @@ export function handleDepositMinted(event: DepositMinted): void {
   const vaultId = event.address.toHexString();
   const foundationId = vaultId + '-' + event.params.groupId.toString();
   const depositId = event.params.id.toString();
-  const claimerId = event.params.claimerId.toString();
+  const claimerId = event.params.claimerId.toHexString();
```

Comment:
Is this change necessary? I made it `toString` on purpose to help with some debugging

Reply:
ohhhhhh I see 👍